llvm/test/tools/llvm-objcopy/ELF/gap-fill-order.test

```diff
@@ -1,16 +1,19 @@
-# RUN: yaml2obj %s >%t
+# RUN: yaml2obj %s -o %t
+
+## In this test, output sections are defined out of
+## order in respect to their load addresses. Verify
+## that gaps are still correctly filled.
```

Comment:
Nit: unnecessarily early line wrapping (there's no strict coding standard for lit test files, but 80 seems like a reasonable value to me, to match other code).

Reply:
Done.